ServerCore/Pages/Pieces/InternMeta.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/InternMeta/{puzzleId}"
+@model ServerCore.Pages.Pieces.InternMetaModel
+
+@{
+    ViewData["Title"] = "Intern Puzzleday Meta";
```

Comment:
Is there any way to actually get to this page? Will we need to add that later?

Reply:
Since part of the URL is the puzzle ID, I'm not sure how to do this. I was assuming we would put a link to it in a puzzle or make a puzzle with a CustomUrl.

Reply:
CustomUrl can do it. That said, for puzzleday, we'd probably end up with a URL in the puzzle PDF since we don't use the puzzle links for it otherwise.

Reply:
Since we don't post PDFs for interns, we're more likely to make aka.ms/PD2019Meta or something.

Reply:
Now that I've got changes to the DB to denote when a page supports this meta, I've got a link to the pieces directly on the Play page.